Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -221,10 +237,12 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
         //let inspiratoBlueDim: UIColor = UIColor(red: 0.0, green: 0.66, blue: 0.875, alpha: 0.25)
```

Comment:
Similarly, there is a problem with setting `numberOfImages` in the `setupCollectionView` method. What happens if the datasource gets a new image? You'd have to reload the whole view controller to update this, right? You should instead do something like:

```swift
public var numberOfImages: Int {
    return dataSource?.numberOfImagesInGallery(self) ?? 0
}
```